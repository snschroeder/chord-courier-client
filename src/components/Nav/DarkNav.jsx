import React from 'react';
import { Link } from 'react-router-dom';
import './dark-nav.css';

export default function DarkNav() {
  return (
    <nav className="dark-nav-bar">
      <ul className="dark-link-list">
        <Link to="/search" style={{ textDecoration: 'none' }}>
          <li className="dark-nav-link">Progression Randomizer</li>
        </Link>
        <Link to="/full-search" style={{ textDecoration: 'none' }}>
          <li className="dark-nav-link">Chords by Key</li>
        </Link>
        <Link to="/about" style={{ textDecoration: 'none' }}>
          <li className="dark-nav-link">About</li>
        </Link>
      </ul>
    </nav>
  );
}
