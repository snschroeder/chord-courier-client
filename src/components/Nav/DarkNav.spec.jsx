/* eslint-disable no-undef */
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { BrowserRouter } from 'react-router-dom';
import DarkNav from './DarkNav';

describe('DarkNav component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <DarkNav />
      </BrowserRouter>,
      div,
    );
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders the header correctly', () => {
    const wrapper = shallow(<DarkNav />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
