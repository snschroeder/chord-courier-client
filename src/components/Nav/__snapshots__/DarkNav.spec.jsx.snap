// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DarkNav component renders the header correctly 1`] = `
<nav
  className="dark-nav-bar"
>
  <ul
    className="dark-link-list"
  >
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/search"
    >
      <li
        className="dark-nav-link"
      >
        Progression Randomizer
      </li>
    </Link>
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/full-search"
    >
      <li
        className="dark-nav-link"
      >
        Chords by Key
      </li>
    </Link>
    <Link
      style={
        Object {
          "textDecoration": "none",
        }
      }
      to="/about"
    >
      <li
        className="dark-nav-link"
      >
        About
      </li>
    </Link>
  </ul>
</nav>
`;
